site/app/templates/admin/users/UserForm.twig

```diff
@@ -109,7 +109,7 @@
     </script>
 {% endblock %}
 {% block form %}
-    <form method="post" action="{{ return_url_update_grader }}" onchange="userFormChange()">
+    <form method="post" action="{{ core.buildUrl({'component': 'admin', 'page': 'users', 'action': action}) }}" onchange="userFormChange()">
```

Comment:
I guess the `return_url_update_grader` wasn't being passed to this twig file at all. You might want to shift it in the View file